src/main/java/lotto/StringAddCalculator.java

```diff
@@ -0,0 +1,43 @@
+package lotto;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+	private static String DEFAULT_DELIMITER = ",|:";
+
+	public static int splitAndSum(String inputText) {
+		inputText = Optional.ofNullable(inputText).map(String::valueOf).orElse("");
+		Matcher m = Pattern.compile("//(.)\n(.*)").matcher(inputText);
+		if (m.find()) {
+			return sum(m.group(2).split(m.group(1)));
```

Comment:
```suggestion
			String delimiter = m.group(1);
			String expression = m.group(2);
			return sum(expression.split(delimiter));
```

한줄에 표현식을 다 써버리면 코드 읽기가 어렵고 유지보수가 어려워집니다